client/landing/jetpack-cloud/sections/backups/utils.js

```diff
@@ -0,0 +1,11 @@
+export const getBackupAttemptsForDate = ( logs, date ) => ( {
+	complete: logs.filter(
+		item =>
+			'rewind__backup_complete_full' === item.activityName &&
+			item.activityDate.split( 'T' )[ 0 ] === date
```

Comment:
this can fail to get the produce the correct result since `date` is in the local timezone and backups are in UTC ( i.e. a backup occurred "today" in my timezone but it doesn't show up ). However I don't think it is a blocker for this PR, we should follow up with a dedicated PR.

Reply:
Agree there are some time issues to work on, also agree it's not a blocker right now.